custom.css

```diff
@@ -396,25 +330,252 @@ textarea {
+    background-color: #fff;
+  }
+
+  /* Header and toolbar section */
+  .container1 {
+    background-color: #fff;
+    position: relative;
+    height: 916px;
+    width: 100%;
+    border-bottom-left-radius: 100px;
+    background-image: url(./images/Desk_Head.png);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: center;
+  }
+
+  .menu-open,
+  .mobile_skill {
+    display: none;
+  }
+
+  .desktop_skill {
+    display: block;
+  }
+
+  .header-details > a {
+    margin-right: 20px;
+  }
+
+  .menu-list {
+    display: flex;
+    list-style-type: none;
+    gap: 32px;
+  }
+
+  .menu-list a:link,
+  .menu-list a:visited {
+    text-decoration: none;
+    color: #344563;
+  }
+
+  .menu-list a:hover,
+  .menu-list a:active {
+    text-decoration: none;
+    color: #344563;
+  }
+
+  .logo {
+    padding: 10px 12px;
+    font-size: 20px;
+    font-weight: 700;
+  }
+
+  span {
+    display: block;
+  }
+
+  .header-banner {
+    display: flex;
+    flex-direction: column;
+    width: 63%;
+    margin: 0 auto;
+  }
+
+  .header-banner-title {
+    margin-top: 289px;
+    width: 920px;
+    font-size: 48px;
+  }
+
+  .header-banner-info {
+    font-weight: 400;
+    font-size: 20px;
+    width: 920px;
+    line-height: 28px;
+  }
+
+  .header-social {
+    margin-top: 12px;
+  }
+
+  .header-head {
+    color: #7f8cff;
+    line-height: 24px;
+    font-size: 16px;
+    font-weight: 500;
+  }
+
+  .header-details {
+    margin-top: 12px;
+  }
+
+  .container {
+    display: none;
+  }
+
+  .container3 {
+    display: block;
+    width: 75%;
+    height: 100%;
+    margin: 142px auto;
+  }
+
+  .card-container {
+    border-radius: 24px;
+    background-color: #fff;
+    margin-top: 142px;
+    padding: 24px;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    column-gap: 30px;
+  }
+
+  .card-header {
+    margin-top: 20px;
+    color: #172b4d;
+    font-weight: 700;
+    font-size: 40px;
+  }
+
+  .card-content > p {
+    color: #344563;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .category {
+    list-style-type: none;
+    color: #6070ff;
+    margin-top: 16px;
+    display: flex;
+    gap: 8px;
+  }
+
+  .card-content .category {
+    text-decoration: none;
+  }
+
+  .category li {
+    background-color: #ebebff;
+    border-radius: 5px;
+    padding: 4px 12px;
+  }
+
+  .card-sub li {
+    color: #7a869a;
+    font-size: 18px;
+  }
+
+  .card-sub .remove {
+    list-style-type: none;
+    color: #344563;
+    font-weight: 600;
+    font-size: 18px;
+  }
+
+  .card-sub {
+    margin-top: 12px;
+    display: flex;
+    justify-content: flex-start;
+    gap: 30px;
+  }
+
+  .about-banner {
+    background-color: #fff;
+    width: 100%;
+    height: 924px;
+    margin: 0 auto;
+    border-top-right-radius: 80px;
+  }
+
+  .about-grid {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    column-gap: 24px;
+    align-items: center;
+    margin: 0 142px;
+  }
```

Comment:
- Kindly, in the about section, the left one takes 40% and the right one the remaining 60% of the page's width, and the contents of each of them are aligned to the right, however, you used `grid-template-columns: 1fr 1fr;` which means 50% for both, please update it to stick the Figma design 👍 
### your design 👇
<img width="1111" alt="Screenshot 2023-03-31 at 11 53 38 AM" src="https://user-images.githubusercontent.com/52010789/229074783-848e4411-9172-42fe-b4b7-10e77c37db9b.png">
### Figma design 👇
<img width="818" alt="Screenshot 2023-03-31 at 11 55 33 AM" src="https://user-images.githubusercontent.com/52010789/229075021-c8eae2a9-b7c1-40b9-bf64-832672bf41d7.png">